go-controller/pkg/node/port_claim.go

```diff
@@ -120,61 +145,61 @@ func updateServicePortClaim(oldSvc, newSvc *kapi.Service) []error {
 	return errors
 }
 
-func (p *portClaimWatcher) open(port int32, protocol kapi.Protocol, svc *kapi.Service) error {
-	klog.V(5).Infof("Opening socket for service: %s/%s and port: %v", svc.Namespace, svc.Name, port)
```

Comment:
Can you keep this log line (or something similar)?

Reply:
Done